Src/Plugins/Security/Security.SharedStrings.txt

```diff
@@ -1 +1,24 @@
-﻿$CSourceFiles=(?i)\.(c|cpp|cxx)$
+﻿#
+# This file contains key values pairs which are used to expand variable references in
+# the JSON-based regex pattern definition files. This is useful both to share common
+# regex patterns across multiple rules but also to avoid the complexities of escaping
+# JSON string literals. The strings below are directly testable in web sites such as 
+# reg101.com.
+#
+# Shared strings can be assembled even within this file as long as any shared pattern
+# in a strings only contains variables which are defined earlier in the file. So, 
+# a pattern like this is ok (but these lines in reverse order are not).
+#
+# $MyExample=[a-z]
+# $MyExandedExample=($MyExample|[0-9])
+#
+# Blank lines are ignored when parsing, as are lines that begin with a '#' character.
+# Leading spaces are trimmed at parse time, allowing for indentation.
+
+# Common patterns for file allow and deny lists.
+#
+  $CSourceFiles=(?i)\.(c|cpp|cxx)$
+  $BinaryFiles=(?i)\.(bmp|dll|exe|gif|jpe?g|lock|pack|png|psd|tar\.gz|tiff?|ttf|xcf|zip)$
```

Comment:
>pack [](start = 49, length = 4)

added pack files. also fixed the tiff? query and minimized jpg|jpeg to a single expression.

tiff? was previously rendered as tif? which matches *.ti and *.tiff. I think we want to match *.tif and *.tiff instead.